React/todoApp/src/components/Form.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./Form.module.css";
+
+export default function Form({ setTodos }) {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const value = e.target.todo.value.trim();
+    if (!value) return;
```

Comment:
1. always use new line between conditional blocks and statements

2. One line or not, let's use `{` blocks when using `if` 
```suggestion
    const value = e.target.todo.value.trim();
    
    if (!value) {
      return
    };
```